packages/backend/src/index.ts

```diff
@@ -14,11 +14,52 @@
  * limitations under the License.
  */
 
+import { TokenManager } from '@backstage/backend-common';
 import { createBackend } from '@backstage/backend-defaults';
-import { createBackendFeatureLoader } from '@backstage/backend-plugin-api';
+import {
+  coreServices,
+  createBackendFeatureLoader,
+  createServiceFactory,
+  createServiceRef,
+} from '@backstage/backend-plugin-api';
+import {
+  DefaultIdentityClient,
+  IdentityApi,
+} from '@backstage/plugin-auth-node';
 
 const backend = createBackend();
 
+backend.add(
```

Comment:
Are these here mostly for demonstration, but no longer used anywhere?

Reply:
They're here mostly by mistake 😁, was just using it to verify what I was putting in the changeset